sdk/digitaltwins/dtdl-parser-generator/parser-src/parser/index.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export { ValueConstraint } from "./type/valueConstraint";
+export { TraversalStatus } from "./enum/traversalStatus";
+export { ModelParsingOption } from "./enum/modelParsingOption";
+export { ParsingError } from "./parsingError";
+
+export { LanguageStringType } from "./type/languageStringType";
+export { EntityKinds } from "./entityKinds";
+export { SchemaInfo } from "./schemaInfo";
+export { SupplementalTypeInfo } from "./supplementalTypeInfo";
+export { ModelParser } from "./modelParser";
+export { ModelDict } from "./modelDict";
+export { SupplementalTypeCollection } from "./supplementalTypeCollection";
+export { EntityInfo } from "./entityInfo";
+export { ArrayInfo } from "./arrayInfo";
```

Comment:
Do we anticipate being able to remove this list from the codebase?  The generator really should not have hard-coded knowledge of the DTDL types (Entity, Array, Boolean, ComplexSchema, etc.).  These should come from the metamodel digest.

Reply:
Yes, absolutely.